UefiCpuPkg/Test/Mock/Library/GoogleTest/MockTimerLib/MockTimerLib.cpp

```diff
@@ -0,0 +1,11 @@
+/** @file MockTimerLib.cpp
+  Google Test mocks for TimerLib
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <GoogleTest/Library/MockTimerLib.h>
+
+MOCK_INTERFACE_DEFINITION (MockTimerLib);
+MOCK_FUNCTION_DEFINITION (MockTimerLib, MicroSecondDelay, 1, EFIAPI);

```

Comment:
APIs are missing

Reply:
Moved to MdePkg and mocked all functions